assets/controller.yaml

```diff
@@ -92,6 +92,7 @@ spec:
             - --leader-election-lease-duration=${LEADER_ELECTION_LEASE_DURATION}
             - --leader-election-renew-deadline=${LEADER_ELECTION_RENEW_DEADLINE}
             - --leader-election-retry-period=${LEADER_ELECTION_RETRY_PERIOD}
+            - --controller-publish-readonly
```

Comment:
@tsmetana do we need to do the same for all the other drivers?

Reply:
So far I only know that this was causing confusion with GCE PD. But it's possible some other drivers would also create writable volumes for ROX PVCs. This needs to be tested. So far I'm aiming at GCE PD where it's known to work unexpectedly (though this option would cause the mount to fail which might be also surprising result for someone...).